.doc_gen/metadata/textract_metadata.yaml

```diff
@@ -0,0 +1,56 @@
+
+#zexi 0.4.0
+textract_AnalyzeDocument:
+  title: Analyze a document using &TEXTRACTlong; and the &AWS; SDK

```

Comment:
&TEXTRACTlong; -> &TEXTRACT;
(short entity in titles)

Reply:
fixed